contracts/KyberProxy.sol

```diff
@@ -0,0 +1,122 @@
+pragma solidity ^0.4.24;
+
+import "./vendors/kyber/KyberNetworkProxy.sol";
+import "./vendors/kyber/ERC20Interface.sol";
+import "./interfaces/TokenConverter.sol";
+import "./utils/Ownable.sol";
+
+contract KyberProxy is TokenConverter, Ownable {
+      
+    mapping(address => mapping(address => KyberNetworkProxy)) public converterOf;
```

Comment:
Kyber network uses multiple converters? This pattern is used in the Bancor network because each converter contract usually only handles 2 or 3 tokens. KyberNetworkProxy may handle all convertions; I am not sure.